src/recovery.rs

```diff
@@ -34,10 +34,32 @@ use std::collections::BTreeMap;
 use crate::Error;
 use crate::Result;
 
+use crate::cc;
 use crate::frame;
 use crate::packet;
 use crate::ranges;
 
+// Provides shortcuts for the following cases:
+//   cc.as_ref().unwrap() -> cc.unwrap_ref()
+//   cc.as_mut().unwrap() -> cc.unwrap_mut()
+trait OptionExt {
```

Comment:
This doesn't seem to be much of a shortcut, and IMO it makes reading the code harder by using custom methods on standard types that normally don't have them. It's also not used consistently AFAICT.

Reply:
OptionExt and all its usage are removed.